ui-tests/src/test/resources/features/integrations/conditional-flow.feature

```diff
@@ -223,20 +225,21 @@ Feature: Conditional flows - content base routing
 
     When fill in values by element data-testid
       | flowconditions-0-condition | ${body.message} == 'Shaco' |
-    And click on the "Done" button
-    Then check that conditional flow step contains 2 flows
-    And check that conditional flow default step is disabled
+    And click on the "Next" button
+    And click on the "Next" button
+    Then check that conditional flow step contains 1 flows
```

Comment:
Is it expected that there are now fewer flows than before?

Reply:
yes, this is intentional